### ISSUE TITLE
How Custom Templates works and how values are fetched in theme builder?

### ISSUE THREAD

**Issue by komalbajaj0007** [pojome/elementor#5061]
- Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!

Hello There,I have a some cool idea, and i want to implement that. So can you please help me?Basically,Can you please share the filename or the code which is doing the work of setting the custom template designed in elementor and how the values are fetched from the database or the custom post type makers like Pods, ACF etcIt will be helpful if you can share some resources. And it can help other members because what i'm building will help other 100%. Thanks
--

- If you have questions or need general support,  Please use:  https://wordpress.org/support/plugin/elementor 

- For help and support from the Elementor community, see: https://www.facebook.com/groups/Elementors/

- To read more about Elementor, check out our documentation: https://docs.elementor.com

- Developers docs are located at: https://developers.elementor.com/

**Comment by bainternet** [comment]
@komalbajaj0007 

We have our Developers docs at https://developers.elementor.com which is currently being updated. Do you have a specific question?

**Comment by komalbajaj0007** [comment]
Yes, i have checked the Docs and not able to find any code which is related to this.

Like basically i want to use Elementor Theme Builder to integrate it with another plugin and basically that plugin also uses templates to create layouts etc. So, if you can share me some code which can allow me to understand the working of Elementor Pro Theme builder functionality of overriding the templates for eg single post,archive etc.
Thanks looking forward to see some helpful example to extend the theme builder integration with other plugins

**Comment by komalbajaj0007** [comment]
@bainternet is there is any reply for my question?

thanks
